cmd/helm/flags.go

```diff
@@ -143,6 +144,9 @@ func (p *postRendererString) Set(val string) error {
 	if val == "" {
 		return nil
 	}
+	if p.options.binaryPath != "" && p.options.binaryPath != val {
+		return errors.Errorf("cannot set post-renderer more than once, given: %s, previous: %s", val, p.options.binaryPath)
```

Comment:
I don't think we need to tell the user the values they supplied. Just stick to the pertinent message:

```suggestion
		return fmt.Errorf("cannot specify --post-renderer flag more than once")
```